lib/unit/test/unit/Cardano/Wallet/Shelley/NetworkSpec.hs

```diff
@@ -226,10 +240,10 @@ observerSpec = sequential $ describe "Observer" $ do
                     (query observer k) `shouldReturn` (Just v)
                     refresh True
                     (query observer k) `shouldReturn` (Just v)
-                    trVar `shouldHaveTraced`
-                        [ MsgWillFetch $ Set.singleton k
-                        , MsgDidFetch $ Map.singleton k v
-                        ]
+                    trVar
+                        `shouldHaveTraced` [ MsgWillFetch $ Set.singleton k
+                                           , MsgDidFetch $ Map.singleton k v
+                                           ]
```

Comment:
Formatting — The indentation introduced here looks dangerously like indentation whose length depends on identifiers. 🤔

Reply:
that is fourmolu, removed the infix for now